test.js

```diff
@@ -1,24 +1,15 @@
-'use strict';
-var test = require('ava');
-var githubMarkdownRender = require('./');
+import test from 'ava';
+import m from './';
 
-test(function (t) {
-	t.plan(2);
+test('promise', async t => {
+	const data = await m('**foo**');
 
-	githubMarkdownRender('**foo**', function (err, data) {
-		t.assert(!err, err);
-		t.assert(data.trim() === '<p><strong>foo</strong></p>', data);
-	});
+	t.true(data.trim() === '<p><strong>foo</strong></p>');
 });
 
-test(function (t) {
-	t.plan(1);
-
-	var stream = githubMarkdownRender.stream();
-
-	stream.on('data', function (data) {
-		t.assert(data.toString().trim() === '<p><strong>foo</strong></p>', data.toString());
-	});
+test('stream', async t => {
+	const stream = await m.stream();
 
+	stream.on('data', data => t.true(data.toString().trim() === '<p><strong>foo</strong></p>'));
```

Comment:
Use `rfpify` or make it `test.cb(...)`. Currently it'll never reach the assertion.